src/content/tutorials/fleet-management/cluster-management/workload-cluster-api-access-for-automation/index.md

```diff
@@ -0,0 +1,145 @@
+---
+linkTitle: Automation access
+title: Access your workload cluster API via automation
+description: Access your cluster API from an automation requires a hands-free way to provide credentials to kubectl or any Kubernetes client. This article explains how to obtain a service account token to use in such a scenario.
+weight: 20
+menu:
+  principal:
+    identifier: tutorials-fleet-management-clusters-automation
+    parent: tutorials-fleet-management-clusters
+user_questions:
+  - How can I use the workload Cluster API in a programmatic way?
+  - How do I grant access to Kubernetes API from my CI system?
+last_review_date: 2024-11-29
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-shield
+---
+
+## Introduction
+
+A `Kubernetes` cluster allows you to interact with its API server securely by using service account tokens. These tokens are primarily used for automation purposes, allowing scripts and applications to perform tasks on the cluster without the need for user intervention. Though they're commonly used by pods to get easy access to the API within the cluster the same method can be reused for automation running out of the cluster.
+
+This documentation entry provides step-by-step instructions on how to access the `Kubernetes` cluster API using service account tokens.
+
+## Prerequisites
+
+Before you proceed, make sure you have the following prerequisites:
+
+1. [Make sure you are logged in the right workload cluster]({{< relref "/tutorials/access-management/authentication/user" >}}).
+2. The [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) command-line tool installed on your local machine.
+3. The necessary permissions to create and manage service accounts within the cluster.
+
+## Create a service account
+
+You need to create a `ServiceAccount` that represents your automation task. Ideally you select the right namespace to live in (you could create a `automation` namespace):
```

Comment:
Stupid question – isn't that provided by rbac-operator?

Reply:
not in WCs AFAIK

Reply:
rbac-operator only does this on org namespaces, meaning only on MCs.